aspnetcore/release-notes/aspnetcore-1.1.md

```diff
@@ -19,7 +19,6 @@ ASP.NET Core 1.1 includes the following new features:
 - [Azure App Service logging provider](xref:fundamentals/logging/index#azure-app-service-provider)
 - [Azure Key Vault configuration provider](xref:security/key-vault-configuration)
 - [Azure and Redis Storage Data Protection Key Repositories](xref:security/data-protection/implementation/key-storage-providers#azure-and-redis)
-- [WebListener Server for Windows](xref:fundamentals/servers/weblistener)
```

Comment:
See earlier comment, especially here in a doc dedicated to 1.x, it could be confusing to drop mention of WebListener altogether.